web/js/components/measure-tool/measure-button.js

```diff
@@ -61,6 +60,13 @@ class MeasureButton extends React.Component {
     const message = isTouchDevice ? mobileHelpMsg : helpMsg;
     const buttonId = 'wv-measure-button';
     const labelText = 'Measure distances & areas';
+    const mobileMeasureButtonStyle = {
+      bottom: '20px',
+      fontSize: '14.3px',
+      height: '44px',
+      margin: '0 0 0 4px',
+      padding: '5.72px 9.1px',
```

Comment:
These came from  `web/scss/mobile.scss`. I just copied and pasted these values since all of the button need the same values but I couldn't set all of their values in the same place. I'm not sure why they were set to fractional values.